src/main/java/org/jabref/gui/shared/SharedDatabaseUIManager.java

```diff
@@ -51,8 +51,6 @@ public SharedDatabaseUIManager(JabRefFrame jabRefFrame) {
     @Subscribe
     public void listen(ConnectionLostEvent connectionLostEvent) {
 
-        jabRefFrame.getDialogService().notify(Localization.lang("Connection lost."));
```

Comment:
At least a log should be put here, shouldn't it? - It's about the [shared SQL database](https://help.jabref.org/en/SQLDatabase)

Reply:
Good idea. I have added that to the place, where the event is actually posted.